docs/bootstrap.sh

```diff
@@ -67,40 +70,90 @@ function release_preview {
   fi
 }
 
-function release {
-  echo_header "docs release"
+function docs_cut_version {
+    echo_header "docs version"
 
-  # If we download cached docs, we may not have netlify CLI in node_modules. Install in case.
-  yarn install
+    if [ -z "$1" ]; then
+        echo "Usage: $1 <version>"
+        exit 1
+    fi
+
+    # Store the current branch to return to later
+    current_branch=$(git rev-parse --abbrev-ref HEAD)
+    echo "Original branch: $current_branch"
+
+    COMMIT_TAG=$1
+    echo "Starting docs versioning for $COMMIT_TAG"
+
+    # Stash any unrelated local changes before starting
+    echo "Stashing local changes"
```

Comment:
stashing in case there are changes to other files in the repo we want to persist